ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -129,8 +128,8 @@ public static String getInstanceName(String host, Integer port) {
    * @param instanceConfig the {@link InstanceConfig} associated with the interested instance.
    * @return the rack id associated with the given instance.
    */
-  static Long getRackId(InstanceConfig instanceConfig) {
-    return Long.valueOf(instanceConfig.getRecord().getSimpleField(RACKID_STR));
+  static String getRackId(InstanceConfig instanceConfig) {
+    return instanceConfig.getRecord().getSimpleField(RACKID_STR);
```

Comment:
not sure why this does not show up as a conflict now that #896 is merged, but please take a look.

Reply:
I think it is conflicting now. I will fix the conflicts, though